frontend/app/components/layouts/protected-layout.tsx

```diff
@@ -64,7 +66,7 @@ function NavigationMenu() {
         </DropdownMenuTrigger>
         <DropdownMenuContent className="w-svw rounded-t-none sm:w-[260px]" sideOffset={0} align="center">
           <DropdownMenuItem asChild className="cursor-pointer">
-            <Link to={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })}>{t('gcweb:header.menu-dashboard.text')}</Link>
+            <Link to={origin?.to ?? ''}>{origin?.text ?? ''}</Link>
           </DropdownMenuItem>
```

Comment:
We shouldn't render a `DropdownMenuItem` with an empty link if the `origin` can't be found:

```suggestion
          {origin && (
            <DropdownMenuItem asChild className="cursor-pointer">
              <Link to={origin.to}>{origin.text}</Link>
            </DropdownMenuItem>
          )}
```